contracts/evoting/types/admin.go

```diff
@@ -44,6 +45,51 @@ func (af AdminForm) Deserialize(ctx serde.Context, data []byte) (serde.Message,
 	return message, nil
 }
 
+// AddAdmin add a new admin to the system.
+func (af *AdminForm) AddAdmin(userID string) error {
```

Comment:
I think you're trying to follow the coding style of the existing code, but please avoid using two-character variable names, if there is no official coding style (there isn't on this project), aiming for readable code is never wrong ;)

Reply:
(also applies for the other parts of the code where this appears - a former work collegue always said to "leave code cleaner than you found it" which I found a very good rule to follow)